pkg/proxy/winkernel/hnsV1.go

```diff
@@ -65,6 +65,11 @@ func (hns hnsV1) getEndpointByID(id string) (*endpointsInfo, error) {
 		macAddress: hnsendpoint.MacAddress,
 		hnsID:      hnsendpoint.Id,
 		hns:        hns,
+
+		// only ready and not terminating endpoints were added to HNS
+		ready:       true,
```

Comment:
We will need these changes in HNSV2.go as well

Reply:
To elaborate all the changes in this file need to be replicated to hnsV2.go .. v2 is the current version in use